MapboxNavigation/NavigationViewController.swift

```diff
@@ -658,23 +622,49 @@ extension NavigationViewController: NavigationServiceDelegate {
         
         //Check to see if we're in a tunnel.
         checkTunnelState(at: location, along: progress)
+        
+        
+        //Pass the message onto our navigation components
+        navigationComponents.forEach { $0.navigationService?(service, didUpdate: progress, with: location, rawLocation: rawLocation) }
```

Comment:
Nit: using `forEach(_:)` here doesn’t really improve readability over a standard `for` loop.